gpkit/tests/t_constraints.py

```diff
@@ -33,6 +33,14 @@ def test_bad_elements(self):
         with self.assertRaises(ValueError):
             _ = Model(x, [v >= ["A", "B"]])
 
+    def test_evalfn(self):
+        x = Variable("x")
+        x2 = Variable("x^2", evalfn=lambda solv: solv[x]**2)
+        m = Model(x, [x >= 2])
+        m.unique_varkeys = set([x2.key])
```

Comment:
mutating `unique_varkeys` seems odd here, wouldn't this happen automatically with a setup method? Should we test that?